src/main/java/Controller.java

```diff
@@ -1658,6 +1696,44 @@ void handleNewOrder(ActionEvent event) {
         }
     }
 
+    @FXML
+    void assignDefaultFileLocation(ActionEvent event) {
+        String pathString = DefaultFileLocation.getText();
+        File temp = new File (pathString);
+        if (temp.exists() && temp != null) {
+            actiontarget.setText("The default file was set to: " + temp.getPath());
+            defaultFL = temp;
+            currentDefaultLocation.setText("The current default file location is: " + defaultFL.getPath());
+            try {
+                File saver = new File(System.getProperty("user.home") + "/DragonSlayer/derbyDB/defaultFilePath.txt");
+                if(saver.createNewFile())
+                {
+                    System.out.println("File created: " + saver.getName());
```

Comment:
If you want to keep each of the System.out.println() for logging purpose it would be better to change it to Log.LogEvent

Reply:
Done and done!